src/securejoin.rs

```diff
@@ -933,9 +939,11 @@ mod tests {
     use std::time::Duration;
 
     #[async_std::test]
-    async fn test_setup_contact() -> Result<()> {
+    async fn test_setup_contact_x() -> Result<()> {
```

Comment:
Why add `_x`?

Reply:
did this rename to be able to only start that test - `cargo test test_setup_contact` starts several other tests that begin with `test_setup_contact`. forgot to rename back then :)